### PR TITLE
Updating to Teams Admin Center Instructions

### PR DESCRIPTION
**Pull request by get-itips**
@scanum Can you kindly ask for review by the article's author?

closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6122

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 1]
```diff
@@ -32,47 +32,47 @@ When you buy **Audio Conferencing** licenses, Microsoft is hosting your audio co
...
 
 ## Steps when you are assigning a new service phone number to your conference bridge
 
 ### Step 1 - Assign the new phone number to your audio conferencing bridge
 
-1. Sign in to Microsoft 365 or Office 365 with your work account.
+![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**
 
-2. Go to **Microsoft 365 admin center** > **Admin centers** > **Teams & Skype** > **Legacy portal** > **Voice** > **Phone numbers**.
+1. In the left navigation, go to **Voice** > **Phone numbers**.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:43]:
```suggestion
1. On the left navigation pane, go to **Voice** > **Phone numbers**.
```

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 2]
```diff
@@ -32,47 +32,47 @@ When you buy **Audio Conferencing** licenses, Microsoft is hosting your audio co
...
 
-3. Select the phone number from the list, and in the Action pane, click **Assign**.
+2. Select the phone number from the list, and click **Edit**.
 
-4. On the **Assign** page, click **Save**.
+3. On the **Edit** page, under **Assigned to** click the dropdown and then click to select **Conference bridge**, and click **Apply**.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:47]:
```suggestion
3. On the **Edit** page, under **Assigned to**, click the dropdown and then select **Conference bridge** > **Apply**.
```

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 3]
```diff
@@ -32,47 +32,47 @@ When you buy **Audio Conferencing** licenses, Microsoft is hosting your audio co
 In addition to the phone numbers already assigned to your conferencing bridge, you can [get additional service numbers](/microsoftteams/getting-service-phone-numbers) (toll and toll-free numbers used for audio conferencing) from other locations, and then assign them to the conferencing bridge so you can expand coverage for your users.
   
 > [!NOTE]
-> To be able to assign/unassign a phone number for a conferencing bridge, the phone number must be a '*service*' number. You can see the type of number it is by navigating to **Voice** > **Phone numbers** in the legacy portal and looking in the **Number Type** column. Microsoft 365 or Office 365 Communications Credits must be set up first in order for users to dial into the bridge on a toll free number.
+> To be able to assign/unassign a phone number for a conferencing bridge, the phone number must be a '*service*' number. You can see the type of number it is by navigating to **Voice** > **Phone numbers** in the Microsoft Teams admin center and looking in the **Number Type** column. Microsoft 365 or Office 365 Communications Credits must be set up first in order for users to dial into the bridge on a toll free number.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:35]:
```suggestion
> To be able to assign/unassign a phone number for a conferencing bridge, the phone number must be a '*service*' number. You can see the type of number it is by navigating to **Voice** > **Phone numbers** in the Microsoft Teams admin center and looking in the **Number Type** column. Microsoft 365 or Office 365 Communications Credits must be set up first in order for users to dial into the bridge on a toll-free number.
```

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 4]
```diff
@@ -32,47 +32,47 @@ When you buy **Audio Conferencing** licenses, Microsoft is hosting your audio co
...
 
 The default phone number of your conference bridge defines the caller ID that will be used when an outbound call is placed by a participant or the organizer from within a meeting.
 
 Only a service toll number can be set as the default number for your conferencing bridge; **service toll-free numbers can't be set as the default number of your conferencing bridge**. If you are assigning a service toll number and you would like to set it as the new default number for your audio conferencing bridge, perform these steps:
 
-1. Sign in to Microsoft 365 or Office 365 with your work account.
+![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**
 
-2. Go to **Microsoft 365 admin center** > **Admin centers** > **Teams & Skype** > **Meetings** > **Conference Bridges**.
+1. In the left navigation, go to **Meetings** > **Conference bridges**.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:57]:
```suggestion
1. On the left navigation pane, go to **Meetings** > **Conference bridges**.
```

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 5]
```diff
@@ -32,47 +32,47 @@ When you buy **Audio Conferencing** licenses, Microsoft is hosting your audio co
...
 ### Step 3 - Change the default phone numbers that are included in the meeting invites of users (optional)
 
 The default phone numbers of a user are the ones that are included on their meeting invites when they schedule a meeting. For more information, including how the default phone numbers are assigned for new users, see [Set the phone numbers included on invites in Microsoft Teams](set-the-phone-numbers-included-on-invites-in-teams.md) or [Set the phone numbers included on invites in Skype for Business Online](/SkypeForBusiness/audio-conferencing-in-office-365/set-the-phone-numbers-included-on-invites).
-  
-1. Sign in with your work or school account.
 
-2. Go to the **Microsoft 365 admin center** > **Admin centers** > **Teams & Skype** > **Legacy portal** > **Audio conferencing** > **Users**, and select the users on the list.
+![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**
+
+1. In the left navigation, go to **Users** and click the Display name of the desired user on the list.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:69]:
```suggestion
1. On the left navigation pane, go to **Users** and click the Display name of the desired user on the list.
```

File: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md [hunk 6]
```diff
@@ -128,13 +128,13 @@ For more information about the Meeting Migration Service, see [Setting up the Me
...
 ### Step 3 - Unassign the old phone number from the audio conferencing bridge
 
-1. Sign in with your work or school account.
+![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**
 
-2. Go to the **Microsoft 365 admin center** > **Admin centers** > **Teams & Skype** > **Legacy portal** > **Voice** > **Phone numbers**.
+1. In the left navigation, go to **Voice** > **Phone numbers**.
 
-3. If the phone number is a toll-free number, select the phone number from the list, and in the Action pane, click **Unassign**. If the phone number is a toll-number, please contact [Microsoft support](https://go.microsoft.com/fwlink/?linkid=2091806) to have the phone number unassigned.
+2. If the phone number is a toll-free number, select the phone number from the list, and click **Release**. If the phone number is a toll-number, please contact [Microsoft support](https://go.microsoft.com/fwlink/?linkid=2091806) to have the phone number unassigned.
```
Comment (JohanFreelancer9) [inline Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md:135]:
```suggestion
2. If the phone number is a toll-free number, select the phone number from the list, and click **Release**. If the phone number is a toll number, please contact [Microsoft support](https://go.microsoft.com/fwlink/?linkid=2091806) to have the phone number unassigned.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions only.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
Can this be reviewed and merged? @scanum @tonysmit Thank you!

**Comment by get-itips** [comment]
Can this one be reviewed and/or merged ? @scanum

**Pull request merged**